projects/optic/src/commands/capture/capture.ts

```diff
@@ -204,47 +204,54 @@ const getCaptureAction =
       denormalize: false,
     });
 
+    const serverDir =
+      captureConfig.server.dir === undefined
+        ? config.root
+        : captureConfig.server.dir;
+    const timeout = captureConfig.server.ready_timeout || 3 * 60 * 1_000; // 3 minutes
+    const readyInterval = captureConfig.server.ready_interval || 1000;
     // start app
     let app: ChildProcessWithoutNullStreams | undefined;
-    if (!options.serverOverride && captureConfig.server.command) {
-      const serverDir =
-        captureConfig.server.dir === undefined
-          ? config.root
-          : captureConfig.server.dir;
-      const timeout = captureConfig.server.ready_timeout || 3 * 60 * 1_000; // 3 minutes
-      const readyInterval = captureConfig.server.ready_interval || 1000;
-
-      try {
-        app = await startApp(
-          captureConfig.server.command,
-          serverDir,
-          captureConfig.server.ready_endpoint,
-          readyInterval,
-          timeout,
-          proxy.targetUrl
-        );
-      } catch (err) {
-        proxy.stop();
-        logger.error(err);
-        process.exitCode = 1;
-        return;
-      }
-    }
 
-    // make requests
     let errors: any[] = [];
     try {
+      let bailout: Bailout = { didBailout: false, promise: Promise.resolve() };
```

Comment:
Had to put this in an object so that:
- We can pass it around to different functions
- When we pass this into functions and mutate the value inside (i.e. didBailout) we need the other functions to see the change (objects are copied by reference, primitives are copied by value)